antarest/study/storage/rawstudy/model/filesystem/config/files.py

```diff
@@ -65,8 +74,40 @@ def build(
             store_new_set=sns,
             archive_input_series=asi,
             enr_modelling=enr_modelling,
+            zip_path=study_path if study_path.suffix == ".zip" else None,
         )
 
+    @staticmethod
+    def _extract_data_from_file(
+        root: Path,
+        inside_root_path: Path,
+        file_type: FileType,
+        multi_ini_keys: Optional[List[str]] = None,
+    ) -> Any:
+        tmp_dir = None
+        if root.suffix == ".zip":
+            from antarest.study.storage.utils import extract_file_to_tmp_dir
```

Comment:
is this because import loop ? better fix the circular import (should be easy since this is just an utils function, you can put it in common/utils)